test/test-express-usage-record.js

```diff
@@ -0,0 +1,186 @@
+          id: 1,
+          remoteMethod: null,
+          method: 'PUT'
+        }, {
+          url: '/api/ModelA/1',
+          model: 'ModelA',
+          id: 1,
+          remoteMethod: null,
+          method: 'POST'
+        }, {
+          url: '/api/ModelC/methodF',
+          model: 'ModelC',
+          id: 1,
+          remoteMethod: 'methodF',
+          method: 'POST'
+        }, {
+          url: '/api/ModelD/1',
+          model: 'ModelA',
+          id: 1,
+          remoteMethod: null,
+          method: 'POST'
+        }, {
+          url: '/api/ModelA/1',
+          model: 'ModelA',
+          id: 1,
+          remoteMethod: null,
+          method: 'DELETE'
+        }, {
+          url: '/api/endpoint1/some/url',
+          model: null,
+          id: null,
+          remoteMethod: null,
+          method: 'POST'
+        }];
+
+        for (var i = 0; i < 500; i++) {
+          var status = Math.random() >= 0.5 ? 200 : 404;
+          var d = new Date();
+          d -= (Math.random() * 23 * 60 * 60 * 1000);
+          var m = endpoints[Math.floor(Math.random() * endpoints.length)];
+
+          updates.push({
+            'timestamp': Number(d),
+            'client': {
+              'address': '127.0.0.' + Math.round(Math.random() * 7),
+              'id': Math.round((Math.random() * 100)),
+              'username': 'a@bcom'
+            },
+            'request': {
+              'method': m.method,
+              'url': m.url
+            },
+            'response': {
+              'status': status,
+              'duration': Math.round((Math.random() * 100))
+            },
+            'process': {
+              'pid': 1235
+            },
+            'data': {
+              'status': status,
+              'url': m.url
+            },
+            'loopback': {
+              modelName: m.model,
+              instanceId: m.id,
+              remoteMethod: m.remoteMethod
+            }
+          });
+        }
+
+        async.each(updates, function(data, callback) {
+          var uInfo = {
+            cmd: 'express:usage-record',
+            record: data,
+          };
+
+          server.handleModelUpdate(1, uInfo, callback);
+        }, tt.end.bind(tt));
+      });
+
+      t.test('print data', function(tt) {
+        server.models.ExpressUsageRecord.dailySummary(function(err, data) {
+          tt.ifError(err);
+          debug(data);
+          tt.assert(Object.keys(data).length > 0, 'Data should be returned');
+          tt.end();
+        });
+      });
+
+      t.test('print data', function(tt) {
+        var hrsAgo5 = new Date() - 5 * 60 * 1000;
+        server.models.ExpressUsageRecord.hourlySummary(
+          'ModelA', hrsAgo5, function(err, data) {
+            tt.ifError(err);
+            debug(data);
+            tt.assert(data.length > 0, 'Data should be returned');
+            tt.end();
+          }
+        );
+      });
+
+      t.test('print data', function(tt) {
```

Comment:
You have three tests cases called "print data", is that intentional? IMO it's a good practice to use unique and descriptive test names (i.e. "hourly summary response", but NOT "print data 2").